plugins/rtmp-services/service-specific/twitch.c

```diff
@@ -54,10 +55,13 @@ static bool load_ingests(const char *json, bool write_file)
 
 	free_ingests();
 
+	rtmps_available = obs_is_output_protocol_registered("RTMPS");
 	for (size_t i = 0; i < count; i++) {
 		json_t *item = json_array_get(ingests, i);
 		json_t *item_name = json_object_get(item, "name");
-		json_t *item_url = json_object_get(item, "url_template");
+		json_t *item_url = json_object_get(
+			item, rtmps_available ? "url_template_secure"
+					      : "url_template");
 		struct ingest ingest = {0};
```

Comment:
I think we should also check `url_template_secure` exists in the JSON in case an ingest has TLS support removed for whatever reason.

Reply:
Sure, easy enough.